package.json

```diff
@@ -178,8 +169,7 @@
     "webpack-merge": "^5.9.0"
   },
   "build": {
-    "productName": "ElectronReact",
-    "appId": "org.erb.ElectronReact",
```

Comment:
what does this appId do? why was this removed?

Reply:
This `appId` is uniquely identifies our app in the world. It is usually the company website in reverse appended by the app name. This is used when we deploy it to any software store like play store, Microsoft store etc. As of now we are not bothered about distribution, and hence removed it. We can add it later when we need to distribute.

Reply:
> This `appId` is uniquely identifies our app in the world. It is usually the company website in reverse appended by the app name. This is used when we deploy it to any software store like play store, Microsoft store etc. As of now we are not bothered about distribution, and hence removed it. We can add it later when we need to distribute.

cool